blockchain/vm/contracts.go

```diff
@@ -60,9 +60,9 @@ type PrecompiledContract interface {
 	Run(input []byte, contract *Contract, evm *EVM) ([]byte, error)
 }
 
-// PrecompiledContractsCypress contains the default set of pre-compiled Klaytn
-// contracts used in the Cypress release.
-var PrecompiledContractsCypress = map[common.Address]PrecompiledContract{
+// PrecompiledContractsConstantinople contains the default set of pre-compiled Klaytn
+// contracts used in the Constantinople release.
```

Comment:
Please add some description about added precompiled contract which we added.

Reply:
```suggestion
// contracts used in the Constantinople release.
// If you want more information about Klaytn's precompiled contracts, 
// please refer https://docs.klaytn.com/smart-contract/precompiled-contracts
```
@KimKyungup How about this?